src/src/components/uuid/tissue_form_components/ruiIntegration.jsx

```diff
@@ -49,26 +49,10 @@ class RUIIntegration extends Component {
 
   componentDidMount() {
     console.log('RUI...', this.props)
-    const runtime_script = document.createElement("script");
-    runtime_script.src = `${process.env.REACT_APP_RUI_BASE_URL}/runtime.js`;
-    runtime_script.async = true;
-    document.body.appendChild(runtime_script);
-    const polyfills_script = document.createElement("script");
-    polyfills_script.src = `${process.env.REACT_APP_RUI_BASE_URL}/polyfills.js`;
-    polyfills_script.async = true;
-    document.body.appendChild(polyfills_script);
-    const main_script = document.createElement("script");
-    main_script.src = `${process.env.REACT_APP_RUI_BASE_URL}/main.js`;
-    main_script.async = true;
-    document.body.appendChild(main_script);
-
-
-    window.dataLayer = window.dataLayer || [];
```

Comment:
FUI google analytics is handled internal to the RUI so this setup code is no longer needed.